rancher2/schema_cluster_v2_rke_config_certificate_rotation.go

```diff
@@ -0,0 +1,27 @@
+package rancher2
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+//Types
+
+func clusterV2RKEConfigRotateCertificatesFields() map[string]*schema.Schema {
+	s := map[string]*schema.Schema{
+		"generation": {
```

Comment:
Same question as above with these fields: what would happen if a cert rotation is done via terraform, and generation isn't set? Or services isn't set? If that would cause a bug in rancher, setting default values would be a good idea here

Reply:
generation is type int64, it will default to 0.  If one is not specified in terraform it will also default to zero.  The generation will not have changed and rancher will take no action.

Reply:
Ah, unspecified generation still results in 0 in rancher. Looks good then